src/stores/map.ts

```diff
@@ -117,6 +120,64 @@ export const useMapStore = defineStore("map", () => {
     }
   }
 
+  /**
+   * Create manual clusters for a given set of geographic spaces.
+   * @param {FeatureCollection} geoJsonSpaces - A GeoJSON FeatureCollection of spaces.
+   * @returns {FeatureCollection} - A new FeatureCollection containing clustered and original features.
+   */
+  function createManualClusters(geoJsonSpaces: FeatureCollection): FeatureCollection {
+    // Group the features by building slug
+    const groupedSpaces = geoJsonSpaces.features.reduce<{ [key: string]: Feature[] }>(
+      (acc, space) => {
+        const buildingSlug = space.properties?.buildingSlug;
+        if (!acc[buildingSlug]) {
+          acc[buildingSlug] = [];
+        }
+        acc[buildingSlug].push(space);
+        return acc;
+      }, {});
+
+    const clusters = Object.entries(groupedSpaces).map(([buildingSlug, spaces]): Feature => {
+      // Calculate the cluster center by averaging the coordinates of all spaces in the cluster
+      const clusterCoordinates = spaces.reduce<[number, number]>(
+        ([sumLon, sumLat], space) => {
+          const coordinates = (space.geometry as Point).coordinates;
+          return [sumLon + coordinates[0], sumLat + coordinates[1]];
+        },
+        [0, 0]
+      );
+
+      const count = spaces.length;
+      const buildingOccupancy = spaces[0].properties?.buildingOccupancy;
+      const buildingSlugModified = spaces[0].properties?.buildingSlugModified;
+
+      // Create a cluster feature with the calculated center and properties
+      return {
+        type: "Feature",
+        properties: {
+          buildingSlugModified,
+          buildingSlug,
+          buildingOccupancy,
+          point_count: count,
```

Comment:
I used the `point_count` and `point_count_abbreviated` because these were the exact namings mapbox is using if you are using the stock clustering functionality. But since we don't use it anymore we can camelCase them and change them in the code.